crates/router/src/connector/aci/transformers.rs

```diff
@@ -15,13 +15,13 @@ pub struct AciAuthType {
     pub entity_id: String,
 }
 
-impl TryFrom<&types::ConnectorAuthType> for AciAuthType {
+impl TryFrom<&common_enums::ConnectorAuthType> for AciAuthType {
     type Error = error_stack::Report<errors::ConnectorError>;
-    fn try_from(item: &types::ConnectorAuthType) -> Result<Self, Self::Error> {
-        if let types::ConnectorAuthType::BodyKey { api_key, key1 } = item {
+    fn try_from(item: &common_enums::ConnectorAuthType) -> Result<Self, Self::Error> {
```

Comment:
This try from method would now return the inner struct of this specific connector. All these methods would go in a separate file called `transformers.rs` in `common_enums`